test/models/plowing_effect_ground_plane/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<model>
+  <name>Ground Plane with plowing effect</name>
+  <version>1.0</version>
+  <sdf version="1.5">model.sdf</sdf>
+
+  <author>
+    <name>Nate Koenig</name>
+    <email>nate@osrfoundation.org</email>
```

Comment:
@adityapande-1995 I'll add you as a co-author on these new models

Reply:
https://github.com/osrf/gazebo/pull/3164/commits/cd9a5869193cbf59ba5e4b311856aa20668e0a20